raiden/transfer/token_network.py

```diff
@@ -29,7 +29,10 @@
     ContractReceiveUpdateTransfer,
 ]
 
-
+# State changes that should be duplicated in case there are two light clients
+DuplicableStateChanges = [
+    ContractReceiveChannelNewBalance
+]
 def subdispatch_to_channel_by_id_and_address(
```

Comment:
i think we can invoke another funcion `subdispatch_to_channel_list `or something, to subdispatch a state change to multiple channels